server/src/main/java/org/opensearch/common/Rounding.java

```diff
@@ -1412,7 +1419,5 @@ public static OptionalLong getInterval(Rounding rounding) {
      * Helper function for checking if the time zone requested for date histogram
      * aggregation is utc or not
      */
-    private static boolean isUTCTimeZone(final ZoneId zoneId) {
-        return "Z".equals(zoneId.getDisplayName(TextStyle.FULL, Locale.ENGLISH));
-    }
+    public abstract boolean isUTC();
```

Comment:
I am wondering if this will break the backward compat check for 2.x? Probably not used as much outside of core